src/store/modules/util/index.ts

```diff
@@ -15,7 +15,9 @@ const utilModule: Module<UtilState, RootState> = {
     partyNames: {},
     cancelReasons: [],
     facilities: {},
-    enumerations: {}
+    enumerations: {},
+    currentFacilityInformation: {},
```

Comment:
We can rename it to facilitiesLatLng and store multiple facility latLng in the format:

```
facilitiesLatLng: {
  facilityId: { latitude: value, longitude: value }
}
```

So when fetching the facility latLng from action we can first check in the state that if the latLng information is available, and only make api call if the information is not available in state.

Making this change will require all the related getter and mutation changes as well.